src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -382,13 +382,13 @@ const MatrixHeader = ({ context, showProjects, project }, reactContext) => {
     return (
         <div className="matrix-header">
             <div className="matrix-header__title">
-                <h1>{context.title}</h1>
-                <div className="matrix-tags">
-                    <MatrixInternalTags context={context} />
-                    {matrixDescription ?
-                        <div className="matrix-description">{matrixDescription}</div>
-                    : null}
+                <div className="matrix-title-badge">
+                    <h1>{context.title}</h1>
+                    {organism ? <MatrixBadges context={context} type={`ReferenceEpigenome-${organism}`} /> : null}
```

Comment:
Why is the "organism" here but not in the previous version? I thought you were just adding badges (I see  "{`ReferenceEpigenome-${organism}").

Reply:
The user could cause an “undefined” if they manually manipulate the query string.